WcaOnRails/spec/models/user_spec.rb

```diff
@@ -353,6 +353,27 @@
     end
   end
 
+  describe "check if registration form sends mail to newly registered user" do
+    it "send mail" do
+      new_user = FactoryBot.create :user, :unconfirmed
+      expect(ActionMailer::Base.deliveries.last.to).to eq [new_user.email]
```

Comment:
This is an interesting way of checking if the email was sent! However, we haven't been checking for emails this way, as it's a little brittle. You already tested that the delivery address and subject are correct in your mailer spec. Instead, we've been using mocks. If you look in this file, you'll see https://github.com/jfly/worldcubeassociation.org/blob/ebb8e956eef9ea57b112333f8ba5835ba0193731/WcaOnRails/spec/models/user_spec.rb#L498, where we check that the right mailer method was called with the correct arguments.